app/workers/publishing_api_notifier.rb

```diff
@@ -5,8 +5,8 @@ def perform(edition_id, update_type = "normal")
     edition = Edition.find(edition_id)
     presenter = PublishedEditionPresenter.new(edition)
     document_for_publishing_api = presenter.render_for_publishing_api(republish: update_type == "republish")
```

Comment:
We don't need the full presenter here, just the content_id which is available from `edition.artefact`.